pages/linux/groups.md

```diff
@@ -0,0 +1,11 @@
+# groups
+
+> Print the groups a user is in.
```

Comment:
Perhaps ....`groups that a user`.... would read better?

Reply:
Do you think "Print group memberships for a user" would be better ?